### ISSUE TITLE
Issue with puppet on debian stretch

### ISSUE THREAD

**Issue by philjonas** [commonsense/conceptnet-puppet#2]
While trying to install conceptnet 5.6 using puppet, I ran into these error messages:
```
Could not autoload puppet/face/module/build: undefined method \`_' for [...]
Could not parse application options: Could not autoload puppet/face/module/build: undefined method `_' for [...] 
```
Which is strange because not only I followed the steps laid out on the readme page but I also installed puppet and facter to see if it would change anything.

Before that, I have managed to install conceptnet 5.5 on the same debian 9.4 stretch environment. While the API would work fine using Docker, it never managed to find the database even when I would point to it using the appropriate environment variable.

Am I missing some configuration steps?

Thanks in advance

**Comment by rspeer** [comment]
As I described briefly in the documentation, there are two reasons I used Puppet 3 for the system configuration:

- Puppet 3.8 is the version that's available in Ubuntu 16.04
- Puppet 4 and later seem to insist on a client-server model that itself is difficult to set up (as it seems you can tell from Debian tripping over its dependencies)

Could you try uninstalling all Puppet dependencies, and using Puppet 3.8 instead? If the Debian command for requesting this particular version is one that would also work on Ubuntu, I could even include this in the setup script.

A quick and dirty fix that could fix your ConceptNet 5.5 problem: find the "conceptnet5_data" volume, which should contain a directory named `/data/conceptnet/psql`, and place an empty file named `done` in that directory.

**Comment by philjonas** [comment]
I ended up choosing the first option. I'm using a fresh VM with Ubuntu 16.04 desktop edition. 
Thank you for your prompt reply.
